src/emmy/expression/analyze.cljc

```diff
@@ -19,6 +19,7 @@
   an expression with respect to an analyzer is therefore effected by a
   round-trip to and from the canonical form."
   (:require [emmy.expression :as x]
+            [emmy.generic :as g]
             [emmy.numsymb :as sym]
             [emmy.util :as u]
             [emmy.value :as v]))
```

Comment:
can this import go away or no?

Reply:
there's a ref. to `g/freeze` and `v/integral?`

Some of the utility level stuff in v/ could move to g/, and then the import of v would occur less often. Willing to consider that